ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Update.scala

```diff
@@ -197,16 +200,19 @@ object Update {
 
   /** Signal that a command submitted via [[WriteService]] was rejected.
     *
-    * See the different [[RejectionReason]] for why a command can be
-    * rejected.
+    * @param cancelled If false, the [[ReadService]]'s deduplication and submission rank guarantees
+    *   apply to this rejection. The participant state implementations should
+    *   strive to set this flag to false as often as possible so that applications
+    *   get better guarantees.
```

Comment:
TODO: add commentary for all parameters, and in particular, refer to the guidance on the error messages given via the Ledger API

Reply:
Added a `TODO (v2)`.